spring-cloud-dataflow-server-local/src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ spring:
       registry:
         populator:
           locations: classpath:META-INF/${binder}-apps.properties
-
+serverStore: redis
```

Comment:
Maybe we move redis or memory into the testing directory so we can at least
test but not ship multiple impls.
M

On Jun 1, 2016, at 8:59 AM, Ilayaperumal Gopinathan <
notifications@github.com> wrote:

In spring-cloud-dataflow-server-local/src/main/resources/application.yml
<https://github.com/spring-cloud/spring-cloud-dataflow/pull/634#discussion_r65354656>
:

> @@ -10,7 +10,7 @@ spring:
>        registry:
>          populator:
>            locations: classpath:META-INF/${binder}-apps.properties
> -
> +serverStore: redis

Yeah, I agree. I thought of having this as temporary until we move away
from redis. But, that is not necessary as we can rely on H2 and use the
default. I will update the PR to move the serverStore property completely.
Also, we need to decide whether we really need the in-memory implementation
of these repositories as they are never used.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/spring-cloud/spring-cloud-dataflow/pull/634/files/2e189a019d922508760c6ac242e322271b013636#r65354656>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAPGqmzlo8bG8_Y9DGUfR-Aez_50s_DMks5qHYIYgaJpZM4IrffL>
.